packages/url_launcher/url_launcher_platform_interface/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 2.1.2
```

Comment:
The CHANGELOG and version change should be reverted per https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates (this is a dev-only change, and thus exempt from both).

Reply:
I think i need to publish a new version for linux plugin_test to pass? https://github.com/flutter/flutter/pull/119968/checks?check_run_id=11180655690

Reply:
Nope, that test is running on a pinned+rolled copy of this repository.